### ISSUE TITLE
Repeate and Loop didn't work as they should

### ISSUE THREAD

**Issue by Hotohori** [FredTungsten/ScriptPlayer#285]
Loop from A to B work only one time, the second time it plays the rest of the video and stops.

Repeat in the playlist, no matter of all or a single file, do the same, repeat one time and then stops and didn't repeat endless.

**Comment by FredTungsten** [comment]
Yeah, some features completely break in combination - I crammed waaaaaay to many special requests in there and now it's a bit unreliable with certain features.

**Comment by skyyeat** [comment]
I want to mention that this bug still exists. The loop function on videos seems to break after 1 or 2 playbacks. If you reset the video to the start on the timeline it plays, then loops again.

"Repeat Single File" does seem to work so long as the general "Repeat" is off though.
